ADAL/src/cache/ADTokenCacheItem+Internal.m

```diff
@@ -34,6 +34,7 @@
 @implementation ADTokenCacheItem (Internal)
 
 #define CHECK_ERROR(_CHECK, _ERR) { if (_CHECK) { if (error) {*error = _ERR;} return; } }
+#define THIRTY_DAYS_IN_SECONDS 2592000
```

Comment:
Make this "30 * 24 * 60 * 60" so it's easier to visually inspect. The compiler will multiply all of that at build time for you.